docs/setup/install.md

```diff
@@ -5,6 +5,8 @@
 Make sure the following conditions are met:
 
 - A Kubernetes cluster with version 1.21 or newer is running. Learn how to [install the Kubernetes tools](https://kubernetes.io/docs/tasks/tools/).
+- Your cluster has at least 1 node with 2+ CPUs and 512+ MB of memory available for the JobSet controller manager Deployment to run on. **NOTE**: On some cloud providers, the default node machine type will not have sufficient resources to run the JobSet controller manager and all the required kube-system pods, so you'll need to use a larger
```

Comment:
In your issue, you mention that you wanted the note be bolded.

In this case, **Note** is bolded and the rest of the text is normal whitespace.

Reply:
Updated